ConfigSpace/conditions.py

```diff
@@ -180,7 +183,8 @@ def _evaluate(self, evaluations):
 
 
 class EqualsCondition(AbstractCondition):
-    def __init__(self, child, parent, value):
+    # ToDo: typeof value?
```

Comment:
Since all hyperparameter values so far are `Union[str, float, int]`, this can be the same.